website/source/api/secret/pki/index.html.md

```diff
@@ -908,6 +920,7 @@ $ curl \
   "data": {
     "allow_any_name": false,
     "allow_ip_sans": true,
+    "allow_uri_sans": true,
```

Comment:
Looks like this was missed in the bool -> slice change. Should be `allowed_uri_sans: "spiffe://*"` or something?

Reply:
Also from below:

> Only valid if the role allows URI SANs (which is the default).

We should probably mention that default here too as it's non-obvious that it's only mentioned in other fields that relate to this...

Reply:
Cool, will get on this first thing tomorrow morning.